### PR TITLE
Add support for overriding the debian package name- to allow picking up the iot release!

### PR DESCRIPTION
**Pull request by NickJLange**
Add support for the playbook installing the iot agent. There is no functionality change outside of this bit of surgery.

### CODE REVIEW

**Comment by albertvaka** [comment]
Could you name this something like `datadog_agent_flavor` and make it work for both apt and rpm? There's no need for this to be only available for apt.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
...
 | `datadog_apt_cache_valid_time`             | Override the default apt cache expiration time (defaults to 1 hour).                                                                                                                                                                                                                                      |
 | `datadog_apt_key_url_new`                  | Override the default URL to Datadog `apt` key (key ID `382E94DE`; the deprecated `datadog_apt_key_url` variable refers to an expired key that's been removed from the role).                                                                                                                             |
+| `datadog_agent_apt_name`                   | Override the default Debian Package for IOT Installations on RPI                                                                                                                                                                                                                                          |
```
Comment (sarina-dd) [inline README.md]:
```suggestion
| `datadog_agent_apt_name`                   | Override the default Debian Package for IOT Installations on RPI.                                                                                                                                                                                                                                          |  
```

**Review by sarina-dd — APPROVED** [review]
Small period addition, 👍 lgtm for docs

File: README.md (hunk 1 above)
Comment (NickJLange) [reply README.md]: Fixed - Thanks!

**Comment by NickJLange** [comment]
Change makes sense. I don't have a rhel box at home to test but the debian changes went through just fine.

**Review by albertvaka — APPROVED** [review]
Looks great! Thanks a lot for making the change for RHEL 🙇 🙇 🙇

**Pull request merged**